mender-orchestrator-manifest/module-artifact-gen/mender-orchestrator-manifest-gen

```diff
@@ -0,0 +1,154 @@
+#!/bin/sh
+
+set -e
+
+show_help() {
+    cat << EOF
+
+Simple tool to generate Mender Artifacts that contain Mender Orchestrator manifests.
+
+Usage: $0 [options] <manifest> [-- [options-for-mender-artifact] ]
+
+    Options: [ -n|artifact-name -t|--device-type -d|--dest-dir --software-name --software-version --software-filesystem -o|--output_path -h|--help ]
```

Comment:
Good point, I forgot these. I removed them since it seems very unnatural to set these properties for a manifest artifact. If you want them, it should be set in the artifacts that the manifest refers to instead.